aspnetcore/blazor/lifecycle.md

```diff
@@ -266,3 +266,66 @@ For more information on the `RenderMode`, see <xref:blazor/hosting-model-configu
 ## Detect when the app is prerendering
 
 [!INCLUDE[](~/includes/blazor-prerendering.md)]
+
+## Cancelable background work
+
+Apps that perform long-running background work may need to cancel or suspend the background work. The following list indicates some of the reasons why background work items might require cancellation:
```

Comment:
We might want to say something to the effect of just because you navigate away, already executing async operations do not get terminated immediately. This includes network calls (HttpClient), calls to the database etc. It's important to ensure that these operations are terminated to conserve resources.